README.md

```diff
@@ -1 +1,136 @@
-# tf_label
\ No newline at end of file
+# tf_label
+
+Terraform module designed to generate consistent label names and tags for resources. Use `tf_label` to implement a strict naming convention. 
```

Comment:
You should make it clear that you need to define one of these for each resource, it is not meant to be used globally for all resources